cmp/0a91342f-3d61-45c0-ae68-4be3073542d5/component.lp

```diff
@@ -0,0 +1,138 @@
+(librepcb_component 0a91342f-3d61-45c0-ae68-4be3073542d5
+ (name "nRF52810-QCxx")
+ (description "Bluetooth 5.2 SoC supporting Bluetooth Low Energy.\n\nQFN32 package variant.")
```

Comment:
Why is the component package specific? Because of the number of pins?

Reply:
Yep, and sometimes different pins being available (or not). I would have merged the symbols if it was possible.